apis/fluent-en_GB/extensions/atrium-api-fluent-en_GB-kotlin_1_3/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/kotlin_1_3/samples/ResultExpectationSamples.kt

```diff
@@ -57,69 +58,85 @@ class ResultExpectationSamples {
-//        }
-//
-//        fails { // because wrong Expectation type expected
-//            expect(failure)
-//                .toBeAFailure<ArithmeticException>() // fails
-//                .messageToContain("parameter")       // not evaluated/reported because toBeAFailure already fails
-//            //                                          use `toBeAFailure<...> { ... }` if you want that all expectations are evaluated
-//        }
-//
-//        fails { // because it was a Success
-//            expect(Result.success(10))
-//                .toBeAFailure<IllegalArgumentException>() // fails
-//                .messageToContain("parameter")            // not evaluated/reported because toBeAFailure already fails
-//            //                                               use `toBeAFailure<...> { ... }` if you want that all expectations are evaluated
-//        }
-//    }
-//
-//    @Test
-//    fun toBeAFailure() {
-//        val errorMessage = "can not divide by zero"
-//        val failure = Result.failure<Int>(ArithmeticException(errorMessage))
-//
-//        expect(failure).toBeAFailure<ArithmeticException> {  // subject within this expectation-group is of type ArithmeticException
-//            messageToContain("by zero")
-//        } // subject here is back to type Result<Int>
-//
-//        fails { // because sub-expectation fails
-//            expect(failure).toBeAFailure<IllegalArgumentException> {
-//                messageToContain("parameter") // fails
-//                message.toStartWith("wrong")  // still evaluated even though messageToContain already fails
-//                //                               use `.toBeAFailure.` if you want a fail fast behaviour
-//            }
-//        }
-//
-//        fails { // because wrong Expectation type expected, but since we use an expectation-group...
-//            expect(failure).toBeAFailure<ArithmeticException> {
-//                messageToContain("parameter") // ...reporting mentions that subject's message was expected `to contain: "parameter"``
-//                //                               use `.toBeAFailure.` if you want a fail fast behaviour
-//            }
-//        }
-//
-//        fails { // because it was a Success, but since we use a block
-//            expect(Result.success(10)).toBeAFailure<IllegalArgumentException> {
-//                messageToContain("parameter") // ...reporting mentions that subject's message was expected `to contain: "parameter"``
-//                //                               use `.toBeAFailure.` if you want a fail fast behaviour
-//            }
-//        }
-//    }
+//    TODO 0.20.0 activate once we have the workaround for #1234 implemented
+
+
+    @Test
+    fun toBeAFailureResultFix() {
+        val message = "wrong argument"
+        val failure = Result.failure<Int>(IllegalArgumentException(message))
+
+
+
+        expect(failure) {
+            toBeAFailure<IllegalArgumentException>()
+        }
+
+
+    }
```

Comment:
Please remove this one again, suffices if we have it in ResultExpecationsSpec, here we only want samples and not regression tests. Also, the samples below already failed due to the bug, so we do not even need an additional regression here.

Reply:
Do you want me to comment out all the tests to the bottom or just the test?